File: app/src/test/java/com/example/megaport/mynews/MultimediumTest.java
package com.example.megaport.mynews;

import com.example.megaport.mynews.Models.Multimedium;
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class MultimediumTest {

    private final String testStr = "test";
    private final Integer testInt = 1;
    private final Multimedium multimedium = new Multimedium();

    @Test
    public void setUrl() {
        multimedium.setUrl(testStr);
        assertEquals("Url getter or setter is invalid", testStr, multimedium.getUrl());
    }

    @Test
    public void setHeight() {
        multimedium.setHeight(testInt);
        assertEquals("Height getter or setter is invalid", testInt, multimedium.getHeight());
    }

    @Test
    public void setWidth() {
        multimedium.setWidth(testInt);
        assertEquals("Width getter or setter is invalid", testInt, multimedium.getWidth());
    }

    @Test
    public void setType() {
        multimedium.setType(testStr);
        assertEquals("Type getter or setter is invalid", testStr, multimedium.getType());
    }
}

File: app/src/test/java/com/example/megaport/mynews/MultimediumTest.kt
package com.example.megaport.mynews

import com.example.megaport.mynews.models.Multimedium
import org.junit.Test
import org.junit.Assert.assertEquals

class MultimediumTest {

    private val testStr = "test"
    private val testInt = 1
    private val multimedium = Multimedium()

    @Test
    fun setUrl() {
        multimedium.url = testStr
        assertEquals("Url getter or setter is invalid", testStr, multimedium.url)
    }

    @Test
    fun setHeight() {
        multimedium.height = testInt
        assertEquals("Height getter or setter is invalid", testInt, multimedium.height)
    }

    @Test
    fun setWidth() {
        multimedium.width = testInt
        assertEquals("Width getter or setter is invalid", testInt, multimedium.width)
    }

    @Test
    fun setType() {
        multimedium.type = testStr
        assertEquals("Type getter or setter is invalid", testStr, multimedium.type)
    }
}